nextcord/application_command.py

```diff
@@ -837,16 +886,13 @@ async def call_invoke_slash(
 
         await self.invoke_slash(interaction, **kwargs)
 
-    # This method is only meant to be called by nextcord.Client
-    async def _send_error(self, interaction: Interaction, error: ApplicationError) -> None:
+    async def dispatch_error(self, interaction: Interaction, error: ApplicationError) -> None:
```

Comment:
Dispatch isn't the correct name here IMO. "invoke" is probably what you want to match the rest of application commands, and "dispatch" typically refers to events/listeners.

Reply:
Renamed. I made it dispatch_error simply because that was the name of the method in ext.commands